core/types/receipt.go

```diff
@@ -117,6 +117,47 @@ func NewReceipt(root []byte, failed bool, cumulativeGasUsed uint64) *Receipt {
 	return r
 }
 
+// DeriveField fills the receipt with their computed fields based on consensus
```

Comment:
```suggestion
// DeriveFields fills the receipt with their computed fields based on consensus
```

Reply:
It definitely derives multiple fields.
I changed it. Thank you!